FoxHound.Client/src/Posts/AddPost.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState, FormEvent, ChangeEvent } from "react";
+import {
+  Container,
+  Paper,
+  Typography,
+  Grid,
+  TextField,
+  Box,
+  Button,
+} from "@material-ui/core";
+import Axios from "axios";
+import PostContentEditor from "./PostContentEditor";
+import PostContentViewer from "./PostContentViewer";
+
+const AddPost: React.FC = () => {
+  const [title, setTitle] = useState<string>("");
+  const [content, setContent] = useState<string>("");
+  const [isSubmitting, setIsSubmitting] = useState<boolean>(false);
+  const [error, setError] = useState<string>("");
+
+  const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    try {
+      setIsSubmitting(true);
+      const response = await Axios.post<number>(
+        "https://localhost:44360/Post/Create",
+        {
+          blogId: 1,
```

Comment:
This will need to be updated one we have navigation and get blog id from URL or something